airbyte-integrations/connectors/source-google-search-console/source_google_search_console/spec.json

```diff
@@ -105,6 +105,12 @@
             }
           }
         ]
+      },
+      "custom_reports": {
+        "order": 4,
+        "type": "string",
+        "title": "Custom Reports (Optional)",
+        "description": "A JSON array describing the custom reports you want to sync from Google Search Console. See <a href=\"https://docs.airbyte.com/integrations/sources/google-search-console#step-2-set-up-the-google-search-console-connector-in-airbyte\">the docs</a> for more information about the exact format you can use to fill out this field."
```

Comment:
Could we validate this JSON string's structure to ensure it's a valid custom report declaration?

Reply:
Done by defining jsonschema